lib/DnD/DragPreview.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { oneOfType, node, string } from 'prop-types';
+import { useDragLayer } from 'react-dnd';
+import { getItemStyles } from './helpers/getItemStyles';
+
+function DragPreview({ children }) {
+  const { initialOffset, currentOffset } = useDragLayer(monitor => ({
```

Comment:
Our `DragPreview` uses the drag layer aspect of `react-dnd`, we just need to set the offsets.

This was actually retrieved from an example used on their site, I just used the idea in a different way to achieve our custom preview 😄